From d689c55cd7ef85593181a1df1703c6636ae0a29c Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Thu, 16 May 2024 10:53:38 +0000
Subject: [PATCH] BootChainDxePrivate: Increase maximum bootchain retry from 3
 to 7

So that it does not interfere with balenaOS boot count

Upstream-status: Inappropriate [configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 Silicon/NVIDIA/Drivers/BootChainDxe/BootChainDxePrivate.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Silicon/NVIDIA/Drivers/BootChainDxe/BootChainDxePrivate.h b/Silicon/NVIDIA/Drivers/BootChainDxe/BootChainDxePrivate.h
index d8847e5b..5022ac49 100644
--- a/Silicon/NVIDIA/Drivers/BootChainDxe/BootChainDxePrivate.h
+++ b/Silicon/NVIDIA/Drivers/BootChainDxe/BootChainDxePrivate.h
@@ -16,7 +16,7 @@
 #include <Protocol/BrBctUpdateProtocol.h>
 
 #define NUM_BOOT_CHAINS             2
-#define BOOT_CHAIN_MAX_RESET_COUNT  3
+#define BOOT_CHAIN_MAX_RESET_COUNT  7
 
 #define STATUS_SUCCESS                          0
 #define STATUS_IN_PROGRESS                      1
-- 
2.17.1

